frontend/pages/NoAccessPage/NoAccessPage.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useEffect } from "react";
```

Comment:
This new page is routed from the main page or the reset password page if the user has no global or team role